templates/styles/mobile.css

```
@@ -0,0 +1,74 @@
+
+  .modal-image img {
+    display: none;
+  }
+
+  .movie-detail {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+    gap: 0.5rem;
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)